lib/web/apiserver.go

```diff
@@ -305,6 +305,12 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	// web context
 	h.GET("/webapi/sites/:site/context", h.WithClusterAuth(h.getUserContext))
 
+	// Database access handlers.
+	h.GET("/webapi/sites/:site/databases", h.WithClusterAuth(h.siteDatabasesGet))
```

Comment:
"site" is obsolete terminology - ok to keep it in handler paths but let's name functions `clusterDatabasesGet` and `clusterKubesGet`.

Reply:
renamed